README.md

```diff
@@ -126,6 +126,16 @@ You can also use more functional methods like `map()`, `andThen()`, `mapEither()
 
 Microya-Kotlin returns a `JsonApiException` when the request is unsuccessful. You can call `.getBody()` after type casting the `jsonApiException` to a `ClientError`
 
+### Step 3b: Multipart File Upload
+
+Microya-Kotlin supports file uploads using multipart requests. To perform multipart request with Microya-kotlin. You'll need to use the `performUploadRequest` function.
```

Comment:
Suggestion: To perform multipart request with Microya-kotlin**,** you'l... There should be a comma instead of a full stop :).